scico/linop/radon_svmbir.py

```diff
@@ -30,39 +30,65 @@
     raise ImportError("Could not import svmbir; please install it.")
 
 
-class ParallelBeamProjector(LinearOperator):
-    r"""Parallel beam Radon transform based on svmbir.
+class TomographicProjector(LinearOperator):
+    r"""Tomographic projector based on svmbir.
 
     Perform tomographic projection of an image at specified angles, using
     the `svmbir <https://github.com/cabouman/svmbir>`_ package. The
     ``is_masked`` option selects whether a valid region for projections
```

Comment:
Wrong markup: should be single quotes for most of the places (other than None and True) where double quotes are currently used.

Reply:
Fixed it.